.github/workflows/reusable-nox.yml

```diff
@@ -35,5 +35,7 @@ jobs:
         run: |
           nox -e clone-core
       - name: "Run nox -e ${{ matrix.session }}"
+        env:
+          session: "${{ matrix.session }}"
         run: |
-          nox -e "${{ matrix.session }}"
+          nox -e "${session}"
```

Comment:
While other places seem justify the env var, this one doesn't, simply because this is coming from own *hardcoded* matrix.  So maybe revert changes to this specific file?

Reply:
Done!

Reply:
Marking this as resolved.